suite-native/module-settings/src/screens/SettingsAnalyticsScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Screen, ScreenHeader } from '@suite-native/navigation';
+import { Box, Text, Switch } from '@suite-native/atoms';
+import { disableAnalyticsThunk, enableAnalyticsThunk } from '@suite-native/analytics';
+import { selectIsAnalyticsEnabled } from '@suite-common/analytics';
+
+export const SettingsAnalyticsScreen = () => {
+    const dispatch = useDispatch();
+    const isAnalyticsEnabled = useSelector(selectIsAnalyticsEnabled);
+
+    const handleAnalyticsChange = (value: boolean) => {
+        if (value) {
+            dispatch(enableAnalyticsThunk());
+            return;
+        }
+        dispatch(disableAnalyticsThunk());
+    };
+
+    return (
+        <Screen>
+            <ScreenHeader title="Analytics" />
+            <Box flexDirection="row" justifyContent="space-between" marginVertical="medium">
```

Comment:
We don't have component for this?

Anyway, this whole row should be clickable, it's horrible UX if you can click only that small switch.

Reply:
fixed